pubspec.yaml

```diff
@@ -11,9 +11,14 @@ environment:
 dependencies:
   flutter:
     sdk: flutter
+  data:
+    path: packages/data
+  domain:
+    path: packages/domain
```

Comment:
commented higher. the presentation has data, data has domain. no need in specifying here all of them